library/src/main/res/values/attrs.xml

```diff
@@ -8,6 +8,7 @@
     <attr name="tsquare_titleTextColor" format="color"/>
     <attr name="tsquare_displayHeader" format="boolean"/>
     <attr name="tsquare_headerTextColor" format="color"/>
+    <attr name="tsquare_monthsReverseOrder" format="boolean"/>
```

Comment:
this isn't really a display attribute in the same sense as the others.  Let's move this to a method on `FluentInitializer`